src/components/WebexRemoteMedia/WebexRemoteMedia.stories.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
it be nice to have audio enabled, disabled as well. and even have them both enabled or disabled. voice update is still considered as part of the user experience change.